libethash-cuda/ethash_cuda_miner_kernel.cu

```diff
@@ -26,7 +26,7 @@
 #endif
 
 __global__ void 
-__launch_bounds__(TPB, BPSM)
+//__launch_bounds__(TPB, BPSM)
```

Comment:
It depends. As long as the card has enough register file, this should increase the performance.